### PR TITLE
Write tests for price queries

### PR DESCRIPTION
**Pull request by manzoorwanijk**
Completes #2106 

## Checklist

* [ ] I have read the documentation relating to systems affected by this pull request, see https://github.com/eventespresso/event-espresso-core/tree/master/docs
* [ ] User input is adequately validated and sanitized
* [ ] all publicly displayed strings are internationalized (usually using `esc_html__()`, see https://codex.wordpress.org/I18n_for_WordPress_Developers)
* [ ] My code is tested.
* [ ] My code follows the Event Espresso code style.
* [ ] My code has proper inline documentation.
* [ ] My code accounts for when the site is in Maintenance Mode (MM2 especially disallows usage of models)

### CODE REVIEW

File: assets/prototype/domain/eventEditor/data/queries/prices/test/useFetchPrices.test.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+			() => {
+				useInitTicketTestCache();
+				return useFetchPrices();
+			},
+			{
+				wrapper,
+			}
+		);
+
+		await waitForNextUpdate(); // wait for response
```
Comment (nicolad) [inline assets/prototype/domain/eventEditor/data/queries/prices/test/useFetchPrices.test.ts]: A bit funny `await wait..`

File: assets/prototype/domain/eventEditor/data/queries/prices/test/useFetchPrices.test.ts [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+		expect(result.current.data).toHaveProperty('espressoPrices');
+
+		// has nodes
+		expect(result.current.data.espressoPrices).toHaveProperty('nodes');
+
+		// nodes is an array with `length` property
+		expect(result.current.data.espressoPrices.nodes).toHaveProperty('length');
+
+		// `nodes` length is positive
+		expect(result.current.data.espressoPrices.nodes.length).toEqual(nodes.length);
```
Comment (nicolad) [inline assets/prototype/domain/eventEditor/data/queries/prices/test/useFetchPrices.test.ts]: A bit too long - `result.current.data.espressoPrices.nodes.length` :)
Not something critical.

File: assets/prototype/domain/eventEditor/data/queries/prices/test/usePriceQueryOptions.test.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+		const wrapper = ApolloMockedProvider();
+		const { result } = renderHook(
+			() => {
+				useInitTicketTestCache();
+				return usePriceQueryOptions();
+			},
+			{ wrapper }
+		);
+
+		expect(result.current.variables.where.ticketIn).toEqual(nodes.map(({ id }) => id));
```
Comment (nicolad) [inline assets/prototype/domain/eventEditor/data/queries/prices/test/usePriceQueryOptions.test.ts]: Maybe those 2 values can be extracted into 2 variable ?

Comment (manzoorwanijk) [reply assets/prototype/domain/eventEditor/data/queries/prices/test/usePriceQueryOptions.test.ts]: Yes, they can be, but I thought to make it single line would look clean 😄

**Pull request merged**